packages/block-library/src/columns/edit.js

```diff
@@ -120,6 +125,10 @@ function ColumnsEditContainer( {
 				</PanelBody>
 			</InspectorControls>
 			<div { ...innerBlocksProps } />
+			<VisuallyHidden as="span" id={ ariaDescribedById }>
+				Use the Left and Right Arrow keys to navigate through the list
+				of columns.
```

Comment:
This will probably need to be translatable
```suggestion
				{ __( 'Use the Left and Right Arrow keys to navigate through the list of columns.' ) }
```